assets/css/lighttheme.css

```diff
@@ -4,6 +4,6 @@ a:active {
 }
 
 a {
-  color: #303030;
+  color: #0093ff;
```

Comment:
Would something like `#606060` be better here? I also think links should be more noticeable, but this makes everything on the page a bright blue, so it's difficult to distinguish important information. Having only some links be this color (only links in the description, for example) would be better here.

Reply:
I'm also not convinced by the color `#0093ff` for all the links, I changed it to `#61809b` that I think achieves a good balance between making the links more noticeable but without being too bright. What do you think? Otherwise I have no problem in leaving it as it was and highlight only the links of certain sections (such as description or comments).

Reply:
I missed this comment before merging. If you'd like to open a new PR or a new issue I'd be happy to continue discussion there.